spec/basic_examples_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "hypothesis"
+
+def expect_failure(&block)
+  expect(&block).to raise_exception(RSpec::Expectations::ExpectationNotMetError)
+end
+
+RSpec.configure do |c|
+  c.include(Hypothesis)
+  c.include(Hypothesis::Providers)
+end
+
+RSpec.describe "some basic hypothesis tests" do
```

Comment:
Note that rspec isn't the only testing library people use. The other really popular one is [Minitest](https://github.com/seattlerb/minitest), which has a different API and a different style of writing complex tests. The main things rspec uses are nested contexts, `let`, and passing metadata to specs, none of which IIRC Minitest supports.